bib/cmd/bootc-image-builder/image_test.go

```diff
@@ -448,3 +449,236 @@ func TestGenPartitionTableDiskCustomizationRunsValidateLayoutConstraints(t *test
+					MinSize: 1 * datasizes.GiB,
+					BtrfsVolumeCustomization: blueprint.BtrfsVolumeCustomization{
+						Subvolumes: []blueprint.BtrfsSubvolumeCustomization{
+							{
+								Mountpoint: "/",
+								Name:       "rootvol",
+							},
+						},
+					},
+				},
+			},
+			5 * datasizes.GiB,
+		},
+		{
+			"btrfs, big root minsize",
+			5 * datasizes.GiB,
+			[]blueprint.PartitionCustomization{
+				{
+					Type:    "btrfs",
+					MinSize: 10 * datasizes.GiB,
+					BtrfsVolumeCustomization: blueprint.BtrfsVolumeCustomization{
+						Subvolumes: []blueprint.BtrfsSubvolumeCustomization{
+							{
+								Mountpoint: "/",
+								Name:       "rootvol",
+							},
+						},
+					},
+				},
+			},
+			10 * datasizes.GiB,
+		},
+		// lvm
+		{
+			"lvm, no root minsize, expands to rootfsMinSize",
+			5 * datasizes.GiB,
+			[]blueprint.PartitionCustomization{
+				{
+					Type:    "lvm",
+					MinSize: 10 * datasizes.GiB,
+					VGCustomization: blueprint.VGCustomization{
+						LogicalVolumes: []blueprint.LVCustomization{
+							{
+								Name:    "varvol",
+								MinSize: 10 * datasizes.GiB,
+								FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+									Mountpoint: "/var",
+									FSType:     "xfs",
+								},
+							},
+						},
+					},
+				},
+			},
+			5 * datasizes.GiB,
+		},
+		{
+			"lvm, small root minsize, expands to rootfsMnSize",
+			5 * datasizes.GiB,
+			[]blueprint.PartitionCustomization{
+				{
+					Type:    "lvm",
+					MinSize: 1 * datasizes.GiB,
+					VGCustomization: blueprint.VGCustomization{
+						LogicalVolumes: []blueprint.LVCustomization{
+							{
+								Name:    "varlv",
+								MinSize: 1 * datasizes.GiB,
+								FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+									Mountpoint: "/",
+									FSType:     "xfs",
+								},
+							},
+						},
+					},
+				},
+			},
+			5 * datasizes.GiB,
+		},
+		{
+			"lvm, big root minsize",
+			5 * datasizes.GiB,
+			[]blueprint.PartitionCustomization{
+				{
+					Type:    "lvm",
+					MinSize: 10 * datasizes.GiB,
+					VGCustomization: blueprint.VGCustomization{
+						LogicalVolumes: []blueprint.LVCustomization{
+							{
+								Name:    "rootlv",
+								MinSize: 10 * datasizes.GiB,
+								FilesystemTypedCustomization: blueprint.FilesystemTypedCustomization{
+									Mountpoint: "/",
+									FSType:     "xfs",
+								},
+							},
+						},
+					},
+				},
+			},
+			10 * datasizes.GiB,
+		},
+	} {
+		t.Run(tc.name, func(t *testing.T) {
+			cnf := &bib.ManifestConfig{
+				Architecture:  arch.FromString("amd64"),
+				RootFSType:    "xfs",
+				RootfsMinsize: tc.rootfsMinSize,
+			}
+			cus := &blueprint.Customizations{
+				Disk: &blueprint.DiskCustomization{
+					Partitions: tc.partitions,
+				},
+			}
+			pt, err := bib.GenPartitionTable(cnf, cus, rng)
+			assert.NoError(t, err)
+
+			var rootSize uint64
+			pt.ForEachMountable(func(mnt disk.Mountable, path []disk.Entity) error {
```

Comment:
This line needs `_ =` to supress the linter. Or a proper error handling. :)

Reply:
Silly me, thank you